docs/toc.md

```diff
@@ -16,6 +16,8 @@
 ### [🔧 Deploying self-contained applications](scenarios/deployment/self-contained-apps.md)
 ### [🔧 Deploying NuGet packages](scenarios/deployment/nuget-packages.md)
 ### [🔧 Deploying ASP.NET web applications](scenarios/deployment/aspnet-applications.md)
+## [Authoring Libraries for .NET Core RC2](scenarios/library-authoring/index.md)
+### [How to manage package dependency versions for RC2](scenarios/library-authoring/pinning-libraries-to-rc2.md)
```

Comment:
Since the file name was changes to "fastening", I think it should be changed here and in index.md too.